portfolio/src/main/webapp/script.js

```diff
@@ -70,12 +70,34 @@ function createListElement(text) {
 
 async function giveSentimentReport() 
 {
-    console.log("This is the script saying hello!\n");
-    console.log("This is the script saying hello!\n");
-    console.log("This is the script saying hello!\n");
 
-    const responseFromServer = await fetch("/sentiment");
-    const analysis = responseFromServer.json();
+    var result = document.getElementById("sentiment-message").value;
+
+    const data = { message: result };
+
+    console.log("\nGetting postResponse!!!!\n");
+    const postResponse = await fetch("/sentiment", {
+                                        method: "POST",
+                                        headers: {
+                                            "Content-Type": "application/json",
+                                        },
+                                        body: JSON.stringify(data), 
+                                        });
+    // .then(response => response.json())
```

Comment:
Remove these comments if not using this?

Reply:
Agreed! Thanks for catching that!